src/vivarium_census_prl_synth_pop/components/immigration.py

```diff
@@ -60,24 +61,25 @@ def setup(self, builder: Builder):
             immigrant_reference_people["census_household_id"]
         )
 
-        self.household_immigrants = non_gq_immigrants[is_household_immigrant]
-        self.household_immigrants_per_time_step = self._immigrants_per_time_step(
-            self.household_immigrants,
-            builder.configuration,
-        )
         self.non_reference_person_immigrants = non_gq_immigrants[~is_household_immigrant]
-        self.non_reference_person_immigrants_per_time_step = self._immigrants_per_time_step(
-            self.non_reference_person_immigrants,
+        self.non_reference_person_immigrants_per_time_step = self._rescale_acs_value(
+            self.non_reference_person_immigrants["person_weight"].sum(),
             builder.configuration,
         )
 
+        self.household_immigrants = non_gq_immigrants[is_household_immigrant]
         # Get the *household* (not person) weights for each household that can immigrate
-        # in a household move, for use in sampling.
-        self.immigrant_households = households_data[
+        # in a household move.
+        self.household_immigrant_households = households_data[
             households_data["census_household_id"].isin(
                 immigrant_reference_people["census_household_id"]
             )
         ]
+        self.households_immigrating_per_time_step = self._rescale_acs_value(
+            self.household_immigrant_households["household_weight"].sum(),
+            builder.configuration,
+        )
+
         # FIXME -- this can go away once state and PUMA are in the households pipeline, because
```

Comment:
Hasn't this been done?

Reply:
Good point. I should have corrected this in #174. I will make another PR.